hbase-server/src/main/java/org/apache/hadoop/hbase/procedure/flush/RegionServerFlushTableProcedureManager.java

```diff
@@ -150,6 +150,11 @@ public Subprocedure buildSubprocedure(String table, String family) {
       throw new IllegalStateException("Failed to figure out if there is region to flush.", e1);
     }
 
+    if (involvedRegions == null && involvedRegions.isEmpty()) {
```

Comment:
It should be `||`, not `&&`

Reply:
This is even better use: `CollectionUtils.isEmpty(involvedRegions)`

Reply:
thx virajjasani, i have modified it